src/Components/Cart/CartPriceDetails.jsx

```diff
@@ -1,30 +1,49 @@
 import React from 'react'
+import { useCart } from '../../Contexts/CartContext'
   
+
+const getPriceDetails = (cartProductList) => {
+  let productsPrice = 0
+  let discount = 0
+  let deliveryCharg = 0
+  cartProductList.map(product => {
+    productsPrice = productsPrice += (product.price * product.quantity)
+    discount = discount + (product.strikePrice * product.quantity) - (product.price * product.quantity)
+    deliveryCharg = deliveryCharg + 20;
+  })
+  deliveryCharg = productsPrice >= 500 ? 0 : deliveryCharg
```

Comment:
instead of writing values of `500` and `20` hardcoded, you can move those values in separate variables, so that when in the near future if you want to change the delivery charge or something, you can do it from there only.

Reply:
ok I will do it